tools/update_required_branch_checks.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Updates the requires status checks for a branch."""
+
+import argparse
+from github import Github
+from typing import List
+
+YOUR_GITHUB_TOKEN = ''
+assert YOUR_GITHUB_TOKEN != '', 'YOUR_GITHUB_TOKEN must be set.'
+
+TARGET_REPO = 'youtube/cobalt'
+
+EXCLUDED_CHECK_PATTERNS = [
+    'feedback/copybara',
+    '_on_device_',
+    'codecov',
+    # Excludes templated check names.
+    '${{'
+]
+
+# Exclude rc_11 and COBALT_9 releases.
+MINIMUM_LTS_RELEASE_NUMBER = 19
+LATEST_LTS_RELEASE_NUMBER = 24
+
+
+def get_protected_branches() -> List[str]:
+  branches = ['main']
+  for i in range(MINIMUM_LTS_RELEASE_NUMBER, LATEST_LTS_RELEASE_NUMBER + 1):
+    branches.append(f'{i}.lts.1+')
+  return branches
+
+
+def initialize_repo_connection():
+  g = Github(YOUR_GITHUB_TOKEN)
+  return g.get_repo(TARGET_REPO)
+
+
+def get_checks_for_branch(repo, branch: str) -> None:
+  prs = repo.get_pulls(
+      state='closed', sort='updated', base=branch, direction='desc')
+
+  latest_pr = None
```

Comment:
optional: `latest_pr = next(pr for pr in prs if pr.merged)`

Reply:
Decided not to do this as I think the StopIteration error is a bit harder to parse (at least for me)